### DISCUSSION TITLE
Is there any way to change only the `fontWeight` of the `<summary>` part of the details block?

### DISCUSSION THREAD

**Discussion by kohheepeace** [WordPress/gutenberg#57921] [Q&A] [answered]
## Problem
As shown in the following video, changes to the font weight of the details block apply to the entire details block, **both `<summary>` part and `paragraph` part.**

https://github.com/WordPress/gutenberg/assets/29557494/5ba12882-9784-431f-9cae-ba5cae0e6682

## Question
Is there any way to change only the `fontWeight` of the `<summary>` part of the details block?

*P.S.
I also want to change `font-size` of only `<summary>` part.

**Comment by t-hamano** [comment, marked as answer]
Unfortunately, as far as I know, it is currently not possible to change the style of only the `summary` element through the block support user interfaces. How about resetting the child block's style instead?

![summary](https://github.com/WordPress/gutenberg/assets/54422211/d49b4407-13ba-41f5-adb5-29b332db9bc4)

Alternatively, the following approaches may be used:

### Define custom css via theme.json

```json
{
	"$schema": "https://schemas.wp.org/trunk/theme.json",
	"version": 2,
	"styles": {
		"blocks": {
			"core/details": {
				"css": "& > summary { font-weight: bold;}"
			}
		}
	}
}
```

### Write CSS via the Global Styles

Site Editor > Global Styles > Blocks > Details > Advanced > Additional CSS

![image](https://github.com/WordPress/gutenberg/assets/54422211/2ddceeaa-c91c-4413-b8ae-51d3496f79b1)

**Reply by kohheepeace** [reply]
> Unfortunately, as far as I know, it is currently not possible to change the style of only the summary element through the block support user interfaces.

I am glad to confirm that it is not possible currently. Thank you very much 👍.

## About the Solution
Thank you also for the solution suggestions. I have come up with a similar approach.

However, I found these solutions somewhat redundant and cumbersome.

Do you think this is a issue to be fixed ?

**Reply by t-hamano** [reply]
I searched for the issue, but couldn't find anything similar. This can be worked on as an enhancement, so I'll migrate it to an issue.

**Reply by kohheepeace** [reply]
Thanks !
